exonum-java-binding/core/rust/integration_tests/tests/executor.rs

```diff
@@ -18,37 +18,37 @@ extern crate java_bindings;
 extern crate lazy_static;
 
 use integration_tests::executor::{
-    check_detached, check_nested_attach, test_concurrent_threads, test_serialized_threads,
+    check_nested_attach, test_concurrent_threads, test_serialized_threads,
     test_single_thread,
 };
 use integration_tests::vm::create_vm_for_tests;
 use java_bindings::jni::JavaVM;
-use java_bindings::DumbExecutor;
+use java_bindings::Executor;
 
 use std::sync::Arc;
 
 lazy_static! {
     pub static ref VM: Arc<JavaVM> = create_vm_for_tests();
-    pub static ref EXECUTOR: DumbExecutor = DumbExecutor::new(VM.clone());
+    pub static ref EXECUTOR: Executor = Executor::new(VM.clone());
 }
 
 #[test]
 fn single_thread() {
-    test_single_thread(&*EXECUTOR);
+    test_single_thread(EXECUTOR.clone());
```

Comment:
Probably not single with the guys below in the same file :upside_down_face: 

I think we can leave it as is, but please inspect these tests to see if they need to be updated or extended.

Reply:
:question:

Reply:
At least `nested_attach` cannot reliably live in the same source file.

Reply:
Actually it can, but it will be unable to check if the thread is detached correctly after the outer-most scope is finished. Other tests a totally fine to live together.